Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/README.md

```diff
@@ -1,28 +1,80 @@
 # Prioritiser
 ## Overview
-- What is this thing in these here files?
+When designing the solution, we realised that in many cases, such as in automated cars, data output from inference classification is fast and largely disorganised. Unlike computers in cars, the communication delay from system to the user can interfere with their response or action, rapid data flow would likely lead to high cognitive load. Hence, the idea of a data prioritise was integrated into the system. This modular component is a heuristic based AI the aims to analyse and understand the items classified by the inference modules. It is key for our proof of concept to understand the way that the visually impaired interact with their environments. We gained this understanding through our own research and through advice from our research partners at UCLIC. The component aims to process all the input data to tailor relevant assistive data for the user. Hence, this component acts as the ”assistant” or core processor of the service.
 
+The system categorises the data into four priorities (Urgent, High, Medium and Low) based on aset of rule levels of the same name. The items are processed individually through levels of decision functions to determine a final priority. This can be seen in 8, focusing on objects size and distance. The Prioritiser has been designed with inheritance/interfacing to allow for modularity allowing future developers to create custom modules for other use cases. In our case the modules were designed to support the Sight++ PoC.Next, object tracking data was integrated into the prioritiser inclusive of speed analysis. It allowed the Prioritiser to understand new and known objects and manage the priority respectively. The prioritiser can now manage obstacle avoidance as well as detection. It does this by first prioritising for objects that may cause collisions and then, prioritising for significant environmental objects the user should be aware of.
```

Comment:
"can be seen in 8" doesn't point to anything, and below "in fig: 9 & 10" (the figures don't have numbers 😄 )

Reply:
done